internal/server/v1/mappers.go

```diff
@@ -101,8 +101,13 @@ func resourceSpecFromProto(spec *entropyv1beta1.ResourceSpec) (*resource.Spec, e
 		deps[key] = value
 	}
 
+	confJSON, err := spec.GetConfigs().MarshalJSON()
```

Comment:
I don't know if this is the best way to get the JSON version of the value here. need to do some exploration on this. But for now, i have done this to make everything work.